src/ImageSharp/PixelFormats/PixelBlenders/PorterDuffFunctions.Generated.cs

```diff
@@ -16,93 +17,93 @@ internal static partial class PorterDuffFunctions
 
 
 
-        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        [MethodImpl(MethodImplOptions.NoInlining)]

```

Comment:
@vpenades @antonfirsov Confirmed that changing this fixes tests in 64bit on Net Framework 4.7.1 and 4.6.2

We'll need to figure out how to create a reduced sample to report.

Reply:
So that's the thing that's been driving me crazy...